src/i18n/strings/de_DE.json

```diff
@@ -2361,5 +2361,8 @@
     "%(brand)s encountered an error during upload of:": "%(brand)s hat einen Fehler festgestellt beim hochladen von:",
     "Use your account to sign in to the latest version of the app at <a />": "Verwende dein Konto um dich an der neusten Version der App anzumelden<a />",
     "We’re excited to announce Riot is now Element!": "Wir freuen uns bekanntzugeben: Riot ist jetzt Element!",
-    "Learn more at <a>element.io/previously-riot</a>": "Erfahre mehr unter <a>element.io/previously-riot</a>"
+    "Learn more at <a>element.io/previously-riot</a>": "Erfahre mehr unter <a>element.io/previously-riot</a>",
+    "Don't show chat effects": "Chat Effekte nicht zeigen",
+    "Sends the given message with confetti": "Sendet die Nachricht mit Konfetti",
+    " sends confetti": " sendet Konfetti"
```

Comment:
We tend to only change the en_EN.json (base) file in version control and then change the rest via the translate platform https://translate.riot.im/ just to weblate doesn't get confused.

Reply:
does that mean that we have to issue a pull request against the sdk only with English strings and wait for the code to be  merged untill we can see that there are missing trnslations for some strings in German?

Reply:
I'm not certain, your approach may be fine but may cause Weblate confusion.

Reply:
I believe we normally avoid committing translated strings directly in the repo because it may conflict with nearby strings changes that other users have entered through Weblate. It can require some manual merge steps to resolve, so likely best to remove these here and only add them via Weblate.

Reply:
Sorry for any confusion: English strings should be added here via `yarn i18n` and CI should fail if this is missed to ensure new English strings are added for translators, but translated (non-English) strings should be added via Weblate.